_inc/idc-notice.js

```diff
@@ -162,14 +167,18 @@
 	 * MC: Will not be prefixed, and will use dashes.
 	 *
 	 * @param eventName string
+	 * @param extraProps object
 	 */
-	function trackAndBumpMCStats( eventName ) {
-		if ( 'undefined' !== eventName && eventName.length ) {
+	function trackAndBumpMCStats( eventName, extraProps ) {
+		if ( 'undefined' === extraProps || typeof extraProps !== 'object' ) {
```

Comment:
`'undefined' === extraProps` should be `'undefined' === typeof extraProps` and the second part should be `'object' !== typeof extraProps`.